__tests__/app.test.js

```diff
@@ -48,8 +49,39 @@ describe("GET /api", () => {
   });
 
   describe("GET /api/articles", () => {
+    test("200: Should return an array of article objects, each containing necessary properties and without a 'body' property", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(Array.isArray(articles)).toBe(true);
```

Comment:
I understand why you are doing this check but it is implicitly an array as we call `forEach` on the next line